contracts/interfaces/IEverlongAdmin.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+interface IEverlongAdmin {
+    /// @notice Thrown when caller is not the admin.
+    error Unauthorized();
+
+    // ╭─────────────────────────────────────────────────────────╮
```

Comment:
To expand on the conversation we had elsewhere about these comments, this is doing a great job of identifying the important sub-components of a contract's interface. In my opinion, these sub-components include all of the following:

1. Events
2. Errors
3. Structs
4. Constants
5. Immutables
6. State Variables
7. Stateful Functions
8. Getter Functions

We can quibble about the last two. I think `Public` and `View` get at the same idea, but I think we might be referring to `public` or `external` when we say `Public`, and we might be referring to `view` or `pure` when we say `View`. That's why I personally like `Stateful` and `Getter`. 

Moving past that, I think having section identifiers for the other components like events, errors, and structs would be good. What do you think?

Reply:
Totally agree that `Stateful` and `Getter` are more descriptive, also agree that events/errors/structs are good as well.